libimage/runtime_test.go

```diff
@@ -93,15 +92,10 @@ func TestTmpdir(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, "/var/tmp", tmpd)
 
-	os.Setenv(tmpStr, "/tmp/test")
+	t.Setenv(tmpStr, "/tmp/test")
```

Comment:
This is too late: `os.Unsetenv` has already clobbered the value, this won’t restore the original.